src/Modules/OrchardCore.Commerce/Settings/CommerceSettingsDisplayDriver.cs

```diff
@@ -60,7 +60,8 @@ public override async Task<IDisplayResult> EditAsync(CommerceSettings section, B
                     .OrderBy(currency => currency.CurrencyIsoCode)
                     .Select(currency => new SelectListItem(
                         currency.CurrencyIsoCode,
-                        $"{currency.CurrencyIsoCode} {currency.Symbol} - {T[currency.EnglishName]}"));
+                        $"{currency.CurrencyIsoCode} {currency.Symbol} - " +
+                        $"{(string.IsNullOrEmpty(currency.EnglishName) ? T["Unspecified"] : T[currency.EnglishName])}"));
```

Comment:
This string interpolation contains a single expression and nothing more.
So you don't need the `$"{ }"`, just the stuff inside it.